__tests__/app.test.js

```diff
@@ -328,3 +349,29 @@ describe("DELETE: /api/comments/:comment_id", () => {
       });
   });
 });
+describe("GET: /api/users", () => {
+  test("returns an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const usersArray = response.body.users;
+        expect(usersArray.length).toBe(4);
+        usersArray.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("should return a 404 if endpoint does not exist ", () => {
+    return request(app)
+      .get("/api/apples")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("path not found");
+      });
+  });
```

Comment:
A test for a none existent path should really be in its own describe block as it's not just specific to the api/users tests